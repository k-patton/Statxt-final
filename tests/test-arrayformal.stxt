int testfunc(int[3] a, string[4] s)
{
	int i;
	for (i = 0; i < 3; i++){
		print(a[i]);
	}
	for (i = 0; i < 4; i++){
		printstr(s[i]);
	}

	return 1;
}

int main()
{
	string[4] t;
	int[3] b;

	t = ["this", "is", "a", "test"];
	b = [6,2,7];

	print(testfunc(b,t));

	return 0;
}