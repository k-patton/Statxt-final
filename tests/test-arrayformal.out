6
2
7
this
is
a
test
1
